tech_owner.rst

```diff
@@ -0,0 +1,94 @@
+Tech owner
+==========
+
+Bij nens is veel vrijheid om met nieuwe technologieen te experimenteren en die in productie te nemen. Dat maakt ons flexibel maar ook kwetsbaar als we niet de verantwoordelijkheid oppakken die bij die vrijheid hoort.
+
+Meestel begint het experimenteren met nieuwe technologie op basis van interesse of skills van 1 persoon. Maar zo gauw die technologie rijp is voor productie is het van belang de kennis van de technologie in de organisatie te borgen.
+
+Om dat te doen moet degene die de technologie in productie wil nemen er voor zorgen dat hij een mede tech owner vindt. Tech owners zijn verantwoordelijk voor de adaptatie en inbedding van de technologie in het bedrijf.
+
+Het is niet de bedoeling dat *iedere* library of dependency een tech owner moet krijgen. Er is een verschil tussen het gebruik van een Javascript helper library als lodash of moment en het virtualiseren van een datacentrum met Mesos.
+
+**TODO:** verantwoordelijkheden tech owner uitschrijven.
+
+
+Common tech
+-----------
+
+Technologie die iedereen verondersteld wordt te beheersen, hiervoor zijn geen tech owners:
+
+    pip, git, buildout, django, postgresql, sphinx, ubuntu, vagrant + lxc, ...
+
+
+Shared tech
+-----------
+
+Technologie die door een substantieel deel van ontwikkelaars wordt gedeeld en niet als risico wordt gezien:
+
+    Redis, Django Rest Framework, PostGIS, Celery + Flower, Factory Boy, Numpy, Pandas, CSS, supervisor, gunicorn, nginx, TileStache, nose, Bootstrap, ...
+
+**TODO:** misschien is het toch verstandigheid om aan enkele van deze techs wat namen te hangen?
```

Comment:
Misschien zit er een verschil tussen "tech die iedereen **indirect** gebruikt en die we daarom vertrouwen" en "tech waar relatief veel mensen direct mee werken"?

In die zin hoeft aan bootstrap geen naam gehangen te worden. Zat mensen die daarmee werken.

Iedereen gebruikt gunicorn indirect en die doet het altijd en dus vertrouwt iedereen het, maar wie weet hoe de nginx+wsgi/gunicorn combi werkt? Misschien is "iedereen gebruikt het en het gaat nooit stuk" wel OK hier.

Op die manier zou je misschien Pandas naar de categorie hieronder willen verplaatsen. Hoe **direct** werken veel mensen hiermee? Of zijn het maar twee personen?

Hm. Ik weet het nog niet met de categoriën. Buildout en setuptools zijn nogal bedrijfskritisch. Iedereen gebruikt het dus da's ok. Maar als er stront aan de knikker is holt iedereen naar één persoon.

Ik weet m'n oorspronkelijke punt nu ook niet meer. Kan je iets met bovenstaande brainstorm? :-)